mineral-ui/mineral-ui
# [Pagination]: New component

### Description
* Add support for size html attribute with htmlSize prop in TextInput
* Export defaultSortComparator in Table
* Add Pagination component
* Add Pagination website examples
* Add  Table website examples for sortable and selectable paginated tables

### Motivation and context
Closes #147 

### Screenshots, videos, or demo, if appropriate

https://147-pagination--mineral-ui.netlify.com/

### How to test
1. `cd mineral-ui && npm start`
2. Navigate to Pagination
3. Confirm all looks and works well
4. Navigate to Table
5. Confirm sortable-paginated and selectable-paginated examples are working appropriately

### Types of changes
- New feature (non-breaking change which adds functionality)
- Other (provide details below)

Other: See details in Description above

### Checklist
* [x] Renders and functions properly in [all supported browsers](https://github.com/mineral-ui/mineral-ui#browser-support) (#753 will fix IE 11 layout bug)

### How does this PR make you feel?
![Pages turning in 3D from an e-reader](https://media.giphy.com/media/tTfBhSONYIqR2/giphy.gif)
